ChangeLog

```diff
@@ -1,7 +1,23 @@
-2016-11-11 - V4.00.00
+2016-11-11 to 2017-March - V4.00.00-alpha
```

Comment:
Please do not change date format: use 2017-03-24 (instead of "2016-11-11 to 2017-March")